tests/service/privacy_request/request_runner_service_test.py

```diff
@@ -70,14 +70,8 @@ def test_start_processing_sets_started_processing_at(
     run_privacy_request_task.delay(privacy_request_status_pending.id).get(
         timeout=PRIVACY_REQUEST_TASK_TIMEOUT
     )
-    _sessionmaker = get_db_session()
-    db = _sessionmaker()
-    assert (
-        PrivacyRequest.get(
-            db=db, id=privacy_request_status_pending.id
-        ).started_processing_at
-        is not None
-    )
+    db.refresh(privacy_request_status_pending)
```

Comment:
IIRC this doesn't work because it's the session that needs to refresh, and not the db object

Reply:
isn't the `db` resource a session, just named `db`?  It seems to be working okay.

Reply:
It is, sorry I'm conflating terms. When I ran something like this before I remember there being a nuance where the test client's session refresh doesn't capture the underlying DB object changes without creating a new session itself. I think we're not seeing it here because we're refreshing the session only to test that the privacy request's status hasn't changed (ie. goes from cancelled to cancelled)

Reply:
OK but this particular line we're talking about does show a change, the `started_processing_at` field changed from None to a datetime

Reply:
I'll add some `updated_at` assertions to these tests to add confidence here

Reply:
new tests passing!